README.md

```diff
@@ -240,30 +241,31 @@ If you use a plugin that you think should be included in this repo, [PR's](https
 
 ## Customizing Highlight Groups
 
-Since Noirbuddy is built on [colorbuddy.nvim](https://github.com/tjdevries/colorbuddy.nvim), you can use its API to customize specific highlight groups as needed:
+Since Noirbuddy is built on [colorbuddy.nvim](https://github.com/tjdevries/colorbuddy.nvim), you can use its API to customize specific highlight groups as needed. To do this, define a function `custom`, where the param `c` is the colorbuddy module:
```

Comment:
Not thrilled with this, but couldn't come up with a cleaner solution.

The thing is that these overrides need to happen after the colorscheme is actually loaded, so the user can't do this at the time of calling `setup`.